bin/hipcc

```diff
@@ -119,7 +119,7 @@ sub delete_temp_dirs {
 #HIP_PLATFORM controls whether to use NVCC or HCC for compilation:
 $HIP_PLATFORM= `$HIP_PATH/bin/hipconfig --platform` // "hcc";
 $HIP_VERSION= `$HIP_PATH/bin/hipconfig --version`;
-($HIP_VERSION_MAJOR, $HIP_VERSION_MINOR, $HIP_VERSION_PATCH) = split(/\./, $HIP_VERSION);
+$HIP_VERSION_PATCH = (split(/\./, $HIP_VERSION))[-1];
 my ($HIP_VERSION_GITDATE, $HIP_VERSION_GITHASH) = split(/-/, $HIP_VERSION_PATCH);
```

Comment:
We don't need any of these lines anymore. We used to split HIP_VERSION into MAJOR, MINOR, PATCH, GITDATE and GITHASH only for setting the preprocessor macros. Since we don't do that anymore, we can remove these lines entirely.